src/ruleset/functions/unusedComponent.ts

```diff
@@ -22,6 +23,11 @@ export const unusedComponent = createRulesetFunction<{ components: Record<string
 
     const results: IFunctionResult[] = [];
     Object.keys(components).forEach(componentType => {
+      // if component type is `securitySchemes` and we operate on AsyncAPI 2.x.x skip validation
```

Comment:
Can you add the `why` here as well? Just to clarify why it is we skip it.

Reply:
Added :)